dbt/include/teradata/macros/columns.sql

```diff
@@ -0,0 +1,29 @@
+{% macro teradata__alter_column_type(relation, column_name, new_column_type) -%}
```

Comment:
I think it would be useful to add a comment that explains why this macro needs an override.